genstudio-mlr-claims-app/src/genstudiopem/web-src/src/components/ClaimsLibraryPicker.tsx

```diff
@@ -24,7 +24,7 @@ export const ClaimsLibraryPicker: React.FC<ClaimsCheckerProps> = ({
 }) => {
   return (
     <Picker
-      label="Select claim library"
+      placeholder="Select Claims Category..."
```

Comment:
The ticket shows "Select from claims", assuming we discussed and want this phrase instead?

Reply:
in the screenshot of the epic it said "select claims category" so I wnt with that